.pre-commit-config.yaml

```diff
@@ -0,0 +1,27 @@
+---
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.1.0
+    hooks:
+      #- id: flake8
```

Comment:
Seems to be some debug info

Reply:
Removed it was to thing about adding flake8 see  #1118